index.ts

```diff
@@ -0,0 +1,19 @@
+function fizzBuzz(n: number) {
+    for(let i = 1;i <= n;i++){
+        const strings = []; // Using a array of strings to save the different parts of the answer
+        //Decided to do this because we can't change a const and I didn't want to keep track of which string we have to print at the end
+        
+        if(i % 3 == 0)
+            strings.push("Fizz");
+
+        if(i % 5 == 0)
+            strings.push("Buzz");
+
+        if(strings.length == 0)
+            console.log(i);
+        else console.log(strings.join(""));
```

Comment:
Some inconsistency here - console.log inside if is on new line, and the one inside else it's on the same line